qa/integration/README.md

```diff
@@ -78,5 +78,16 @@ The integration tests need to be run from MacOS or Linux.  However, the tests ma
 3. Create a corresponding `test_file_input_spec.rb` in `specs` folder and use the `fixtures` object to get all services, config etc. The `.yml` and rspec file has to be the same name for the settings to be picked up. You can start LS inside the tests and assume all external services have already been started.
 4. Write rspec code to validate.
 
+### Logstash Keystore
```

Comment:
@jakelandis maybe better put this explanation into `secret_store_spec.rb`? Not many people will ever think about going to the Readme for documentation on a specific test imo => much more helpful there right?

Reply:
moved comments to spec.